content/en/_index.html

```diff
@@ -40,6 +40,22 @@ <h3 class="section-head">
   </div>
 </div>
 
+<div id="overview" class="text-center">
+  <h3 class="section-head">Trusted by</h3>
```

Comment:
Can you make sure the heading looks the same as the "What is Kubeflow?" and the other headings on the page (with the underline and color, use the same classes).